projects/1.16/assets/chemlib/chemlib/lang/zh_cn.json

```
@@ -1 +1,407 @@
+  "item.chemlib.ingot_zirconium": "锆锭",
+  "item.chemlib.ingot_niobium": "铌锭",
+  "item.chemlib.ingot_molybdenum": "钼锭",
+  "item.chemlib.ingot_technetium": "锝锭",
+  "item.chemlib.ingot_ruthenium": "钌锭",
+  "item.chemlib.ingot_rhodium": "铑锭",
+  "item.chemlib.ingot_palladium": "钯锭",
+  "item.chemlib.ingot_silver": "银锭",
+  "item.chemlib.ingot_cadmium": "镉锭",
+  "item.chemlib.ingot_indium": "铟锭",
+  "item.chemlib.ingot_tin": "锡锭",
+  "item.chemlib.ingot_antimony": "锑锭",
+  "item.chemlib.ingot_tellurium": "碲锭",
+  "item.chemlib.ingot_cesium": "铯锭",
+  "item.chemlib.ingot_barium": "钡锭",
+  "item.chemlib.ingot_lanthanum": "镧锭",
+  "item.chemlib.ingot_cerium": "铈锭",
+  "item.chemlib.ingot_praseodymium": "镨锭",
+  "item.chemlib.ingot_neodymium": "钕锭",
+  "item.chemlib.ingot_promethium": "钷锭",
+  "item.chemlib.ingot_samarium": "钐锭",
+  "item.chemlib.ingot_europium": "铕锭",
+  "item.chemlib.ingot_gadolinium": "钆锭",
+  "item.chemlib.ingot_terbium": "铽锭",
+  "item.chemlib.ingot_dysprosium": "镝锭",
+  "item.chemlib.ingot_holmium": "钬锭",
+  "item.chemlib.ingot_erbium": "铒锭",
+  "item.chemlib.ingot_thulium": "铥锭",
+  "item.chemlib.ingot_ytterbium": "镱锭",
+  "item.chemlib.ingot_lutetium": "镥锭",
+  "item.chemlib.ingot_hafnium": "铪锭",
+  "item.chemlib.ingot_tantalum": "钽锭",
+  "item.chemlib.ingot_tungsten": "钨锭",
+  "item.chemlib.ingot_rhenium": "铼锭",
+  "item.chemlib.ingot_osmium": "锇锭",
+  "item.chemlib.ingot_iridium": "铱锭",
+  "item.chemlib.ingot_platinum": "铂锭",
+  "item.chemlib.ingot_thallium": "铊锭",
+  "item.chemlib.ingot_lead": "铅锭",
+  "item.chemlib.ingot_bismuth": "铋锭",
+  "item.chemlib.ingot_polonium": "钋锭",
+  "item.chemlib.ingot_astatine": "砹锭",
+  "item.chemlib.ingot_francium": "钫锭",
+  "item.chemlib.ingot_radium": "镭锭",
+  "item.chemlib.ingot_actinium": "锕锭",
+  "item.chemlib.ingot_thorium": "钍锭",
+  "item.chemlib.ingot_protactinium": "镤锭",
+  "item.chemlib.ingot_uranium": "铀锭",
+  "item.chemlib.ingot_neptunium": "镎锭",
+  "item.chemlib.ingot_plutonium": "钚锭",
+  "item.chemlib.ingot_americium": "镅锭",
+  "item.chemlib.ingot_curium": "锔锭",
+  "item.chemlib.ingot_berkelium": "锫锭",
+  "item.chemlib.ingot_californium": "锎锭",
+  "item.chemlib.ingot_einsteinium": "锿锭",
+  "item.chemlib.ingot_fermium": "镄锭",
+  "item.chemlib.ingot_mendelevium": "钔锭",
+  "item.chemlib.ingot_nobelium": "锘锭",
+  "item.chemlib.ingot_lawrencium": "铹锭",
+  "item.chemlib.ingot_rutherfordium": "𬬻锭",
+  "item.chemlib.ingot_dubnium": "𬭊锭",
+  "item.chemlib.ingot_seaborgium": "𬭳锭",
+  "item.chemlib.ingot_bohrium": "𬭛锭",
+  "item.chemlib.ingot_hassium": "𬭶锭",
+  "item.chemlib.ingot_meitnerium": "鿏锭",
+  "item.chemlib.ingot_darmstadtium": "𫟼锭",
+  "item.chemlib.ingot_roentgenium": "𬬭锭",
+  "item.chemlib.ingot_copernicium": "鿔锭",
+  "item.chemlib.ingot_nihonium": "鉨锭",
+  "item.chemlib.ingot_flerovium": "𫓧锭",
+  "item.chemlib.ingot_moscovium": "镆锭",
+  "item.chemlib.ingot_livermorium": "𫟷锭",
+  "item.chemlib.ingot_tennessine": "Tennessine锭",
+  "item.chemlib.ingot_oganesson": "Oganesson锭",
```

```suggestion
  "item.chemlib.ingot_tennessine": "[[石田]]锭",
  "item.chemlib.ingot_oganesson": "[[奥气]]锭",
```
https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/1996#discussion_r891195304